packages/ibm-products/src/components/BreadcrumbWithOverflow/BreadcrumbWithOverflow.js

```diff
@@ -356,6 +356,11 @@ BreadcrumbWithOverflow.propTypes = {
        */
       label: PropTypes.node,
 
+      /**
+       * An optional title label for extra long breadcrumbs
+       */
+      shortText: PropTypes.string,
```

Comment:
Since this is explicitly about the breadcrumb label maybe something more like `shortTitle`, `shortLabel`, or even `shortName`?

```suggestion
      shortText: PropTypes.string,
```